### DISCUSSION TITLE
Custom html: Remove footer text

### DISCUSSION THREAD

**Discussion by Hythera** [upptime/upptime#1003] [General] [answered]
How to remove this footer text?
![image](https://github.com/user-attachments/assets/d5ad4644-62b4-49c9-99b7-10f7eec1d74a)

**Comment by ServerDeveloper9447** [comment]
after running the `pages build and deploy` workflow, go to the `gh-pages` branch and then this file: `client/client.d8b4d1ae.js` (for me, it's this file, for you it can have another name, but for sure the file will be in the `client` directory.)
search for the text "footer" and 
![image](https://github.com/user-attachments/assets/bdc300bf-fcbc-48af-9dfd-c3b6da43d254)
like the about, set anything inside the double-quotes next to it.

**Reply by Hythera** [reply]
Thank you

**Comment by AnandChowdhary** [comment, marked as answer]
You can overwrite the text in this line - https://github.com/upptime/status-page/blob/54c2ff5a3d998d525ee4c7e68dc7ce7414d89c33/i18n.yml#L29 using the internationalization feature (https://upptime.js.org/docs/configuration/#internationalization).

**Reply by Hythera** [reply]
Ty
